site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+
+Here is an example FederationDomain with two IDPs configured.
+
+```yaml
+apiVersion: config.supervisor.pinniped.dev/v1alpha1
+kind: FederationDomain
+metadata:
+  name: my-provider
+  # Must be in the same namespace where the Supervisor is installed.
+  namespace: pinniped-supervisor
+spec:
+  issuer: https://my-issuer.example.com/any/path
+  tls:
+    secretName: my-tls-cert-secret
+  # Available identity providers are selected here...
+  identityProviders:
+    - displayName: ActiveDirectory for Admins
+      objectRef:
+        apiGroup: idp.supervisor.pinniped.dev
+        kind: ActiveDirectoryIdentityProvider
+        name: ad-for-admins
+    - displayName: Okta for Developers
+      objectRef:
+        apiGroup: idp.supervisor.pinniped.dev
+        kind: OIDCIdentityProvider
+        name: okta-for-developers
+```
+
+Now users may use either of the above identity providers to authenticate. You can create
+kubeconfigs for both IDPs for each cluster by using `pinniped get kubeconfig` twice for
+each cluster.
+
+## Important consideration when using multiple identity providers: conflicting usernames and group names
+
+When multiple identity providers are configured onto a FederationDomain, then a user may use any of those
+providers to authenticate to that FederationDomain. Since an identity in Kubernetes is simply a username string
+and a list of group name strings, it is very important to consider what will happen if two users each authenticate
+to a FederationDomain using different identity providers.
+
+1. If two users are assigned the same username string as a result of authenticating, then those two users will be
+   considered the *same user* by Kubernetes, regardless of which IDP they came from.
+2. If two users are assigned the same group name string as a result of authenticating, then those two users will be
+   considered to *both belong to same group* by Kubernetes, regardless of which IDP they came from.
+
+This may or may not be desirable. If the two identity providers are intended to represent distinct, non-overlapping
+sets of users, then it is not desirable for their usernames and group names to conflict by being identical.
+On the other hand, if two identity providers contain the same sets of users, and when those users authenticate
+then your intention is that they are the same identity, then it may be desirable.
+
+Let's consider several example use cases.
+- Imagine two IDPs in which usernames and group names are assigned to users by their administrators
+  separately, with no coordination between them. In this case, a user named "ryan" from one IDP is probably not
+  the same human as the user named "ryan" from the other IDP. The group named "admins" from one IDP may
+  have a totally different intended meaning then the group named "admins" from the other IDP.
+  In this case, it is not desirable for usernames and group names from one IDP to conflict with the usernames
```

Comment:
Yeah, in the future breaking this down into several pages so we can add a menu item for `disambiguation` as an easy to find feature would be nice.

Reply:
Let's consider reorganizing this new page and the pre-existing "Required Configuration" page in a future PR. For this PR, I just want to get the information about how to use the new features onto the site.